### PR TITLE
Feat : sendSeen

### PR DESCRIPTION
**Pull request by sahalMoidu**

### CODE REVIEW

**Comment by pedroslopez** [comment]
I don't think we should mix sending a message with marking a conversation as seen. Maybe separating sendSeen to a function in the Chat model would be more flexible. What do you think?

**Comment by sahalMoidu** [comment]
If you think that is better we can do it that way. I added it in the send message as we were replying to messages without seeing it.

**Comment by sahalMoidu** [comment]
@pedroslopez  Would the chat model or the message model be the best place?

**Comment by pedroslopez** [comment]
> @pedroslopez Would the chat model or the message model be the best place?

@sahalMoidu I would say the chat, because it's the _chat_ that you mark as seen.

File: src/util/Injected.js [hunk 1]
```diff
@@ -33,6 +34,17 @@ exports.LoadUtils = () => {
             throw 'The number provided is not a registered whatsapp user';
         return result.jid;
     };
+    window.WWebJS.sendSeen = async (chatId) => {
+        let chat = window.Store.Chat.get(chatId);
+        if (chat !== undefined) {
+            if (chat.getLastMsgKeyForAction === undefined)
```
[the remark this answers is not in the document]
Comment (sahalMoidu) [reply src/util/Injected.js]: It is a workaround. getLastMsgKeyForAction() is called internally inside the  sendseen. And in my test this functions were undefined and hence throwing error.

[the remark this answers is not in the document]
Comment (pedroslopez) [reply src/util/Injected.js]: Do you know when the function would be undefined? When I was taking a look it seemed like it was defined.

[the remark this answers is not in the document]
Comment (sahalMoidu) [reply src/util/Injected.js]: Almost all the times I had it undefined. I didn’t figure out the real cause. 
Could you run some tests

[the remark this answers is not in the document]
Comment (pedroslopez) [reply src/util/Injected.js]: I've been testing and I still have not experienced this issue... Can you try again repeating your same steps for testing and verifying if it does occur?

[the remark this answers is not in the document]
Comment (pedroslopez) [reply src/util/Injected.js]: I'm removing this for now. If an issue occurs later we need to identify when this occurs and introduce this back in if necessary.

**Pull request merged**